package.json

```diff
@@ -35,28 +33,27 @@
     "version": "yarn format && git add -A src",
     "postversion": "git push origin HEAD && git push --tags",
     "yalc:publish": "yarn build && yalc publish --push",
-    "argument": "echo \"The value of --foo is ${file} end of script\"",
-    "test": "yarn install:warp && jest ${file} && yarn remove:warp",
-    "test:unit": "file=./src/__tests__/unit yarn test",
-    "test:unit:cache": "file=./src/__tests__/unit/cache-leveldb.test.ts yarn test",
-    "test:unit:cache:real": "file=./src/__tests__/unit/cache-leveldb-real-data.test.ts yarn test",
-    "test:integration": "file=./src/__tests__/integration yarn test",
-    "test:integration:basic": "file=./src/__tests__/integration/basic yarn test",
-    "test:integration:basic:load": "--silent=false --detectOpenHandles file=./src/__tests__/integration/basic/contract-loading.test.ts yarn test",
-    "test:integration:basic:arweave": "file=./src/__tests__/integration/basic/arweave-transactions-loading yarn test",
-    "test:integration:internal-writes": "file=./src/__tests__/integration/internal-writes yarn test",
-    "test:integration:wasm": "file=./src/__tests__/integration/wasm yarn test",
-    "test:regression": "yarn install:warp && node ./node_modules/.bin/jest ./src/__tests__/regression && yarn remove:warp",
+    "test": "jest",
```

Comment:
hmm, makes no sense to me now :-) I guess it should be removed as well, since all the 'test...' runs use the jest directly.

Reply:
this allows to run all the tests at once, which we never do..., but we could!
so I don't know, I can remove it, but I think that this line doesn't do any harm...